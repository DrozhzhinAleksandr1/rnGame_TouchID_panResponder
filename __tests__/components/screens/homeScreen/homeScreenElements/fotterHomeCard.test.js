import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { FooterHomeCard } from '../../../../../app/components/screens/homeScreen/homeScreenElement/footerHomeCard';

it('create snapshot FooterHomeCard', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<FooterHomeCard isSessionOpened={true} isInfoOpened={true} isGraphicOpened={true} btnDetailsText={'Детализация'} isQuotesFavorite={true}/>);
    expect(toolbarSnapshot).toMatchSnapshot();
});