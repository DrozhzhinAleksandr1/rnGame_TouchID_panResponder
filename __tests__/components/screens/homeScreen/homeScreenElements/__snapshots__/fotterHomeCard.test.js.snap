// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot FooterHomeCard 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#1D2E3E",
      "flexDirection": "row",
      "height": 50,
      "justifyContent": "space-between",
      "paddingHorizontal": 10,
      "width": "100%",
    }
  }
>
  <ClockGreen />
  <Component
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <FooterBtnInfo
      isActive={true}
    />
    <FooterBtnGraph
      isActive={true}
    />
    <FooterBtnDetails
      btnDetailsText="Детализация"
    />
  </Component>
  <StarGreen />
</Component>
`;
